### ISSUE TITLE
[css-2018] CSS Snapshot 2018

### ISSUE THREAD

**Issue by svgeesus** [w3c/csswg-drafts#2281]
Looking at snapshot 2017, 

A) these items were in the _rough interop_ bucket, should any get promoted to the main list? Or indeed demoted or dropped.

> CSS Transitions Level 1 [CSS3-TRANSITIONS] and CSS Animations Level 1 [CSS3-ANIMATIONS]
> CSS Flexible Box Module Level 1 [CSS-FLEXBOX-1]
> CSS Transforms Level 1 [CSS3-TRANSFORMS]
> CSS Custom Properties for Cascading Variables Module Level 1 [CSS-VARIABLES-1]
> CSS Text Module Level 3 [CSS-TEXT-3]

B) these are in the _fairly stable_ bucket, should any get promoted to _rough interop_?  Or indeed demoted or dropped.

> CSS Counter Styles Level 3 [CSS-COUNTER-STYLES-3]
> CSS Masking Level 1 [CSS-MASKING-1]
> CSS Shapes Module Level 1 [CSS-SHAPES-1]
> CSS Text Decoration Level 3 [CSS-TEXT-DECOR-3]
> CSS Will Change Level 1 [CSS-WILL-CHANGE-1]
> CSS Speech Module Level 1 [CSS3-SPEECH]
> CSS Box Alignment Module Level 3 [CSS-ALIGN-3]
> CSS Grid Layout Module Level 1 [CSS-GRID-1]
> Filter Effects Module Level 1 [FILTER-EFFECTS-1]
> CSS Fragmentation Module Level 3 [CSS-BREAK-3]

C) Are there any specs, unlisted in Snapshot 2017, which should be added to Snapshot 2018?

D) If the answers to the above are No, No and No then I believe we should set expectations and state that Snapshot 2017 applies to 2018 too. Otherwise, we should publish a Snapshot 2018 over the next few weeks.

**Comment by svgeesus** [comment]
In my opinion, there is a good case for moving Flexbox 1 and Variables 1 out of _rough interop_ into the main body of the snapshot.

I think that Grid 1 and Will Change 1 should move up to _rough interop_. CSS Speech should be dropped from the snapshot.

Interested to hear opinions in favor of other modules (or against my suggestions).

**Comment by frivoal** [comment]
@svgeesus I agree with what you said. I would also add css-contain, MQ4, and css-scroll-snap to the "have completed design work, and are fairly stable, but have not received much testing and implementation experience" list.

**Comment by svgeesus** [comment]
https://github.com/w3c/csswg-drafts/issues/1139 and https://github.com/w3c/csswg-drafts/issues/1867 also relevant to the creation of snapshot 2018

**Comment by dbaron** [comment]
It's worth looking at testsuite data, but some thoughts in addition to the previous comments are:
* maybe filter effects could get promoted to "rough interop"
* it's possible that grid could get promoted directly to the main list
* css-timing-1 should be included with animations and transitions since it was split from them
* things worth considering for the "fairly stable" list might be:
  * fonts-4
  * font-loading-3
  * ruby-3
  * color-4
  * cascade-4

**Comment by dbaron** [comment]
Also worth linking to the discussion on modules for inclusion in the 2017 snapshot: #826.

**Comment by frivoal** [comment]
This should be fixed as well before republishing: https://github.com/w3c/csswg-drafts/issues/1007 (ping @tabatkins )

**Comment by frivoal** [comment]
I have made a tentative Pull Request based on this discussion:
https://github.com/w3c/csswg-drafts/pull/2389

I have included all the suggestions in this github issue, except for:
* fonts-4
* font-loading-3
* ruby-3
* color-4

due to the number of open issues. It's possible that many of these issues could be closed easily, or that the part of the spec with issues could be pushed to a next level leaving us with a stable core, but I didn't feel too comfortable calling specs with tens of open issues "fairly stable", even though I do agree that these specs shouldn't be too far off.

Feedback appreciated.

**Comment by svgeesus** [comment]
I checked the open issues on those specs and agree they are not ready for inclusion in this snapshot. The PR looks good and i reviewed it (but to allow others to express opinions, did not (yet) approve it). Let's agree on this by this week's call?
